import.go

```diff
@@ -91,6 +91,13 @@ func importBuilderFromImage(store storage.Store, options ImportFromImageOptions)
 	}
 	img, err := is.Transport.GetStoreImage(store, ref)
 	if err != nil {
+		img2, err2 := store.Image(options.Image)
```

Comment:
It'd create a new variable in the scope of the block, leaving the `err` defined on line 92 unchanged.  We want to clear that one, though.  The "validate" make target would also fail because it'd trigger a warning from the 'vetshadow' linter.